From 40f0f72a432f3f98d220bc7410aed141af00ba80 Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Tue, 7 Sep 2021 11:15:10 +0800
Subject: [PATCH 19/19] config_tools: add the PTM tag in scenario xml file

add the PTM tag in industry.xml for adl-s-crb platform.

Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/industry.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/misc/config_tools/data/adl-s-crb/industry.xml b/misc/config_tools/data/adl-s-crb/industry.xml
index 63ea3cae5..ebe9514ee 100644
--- a/misc/config_tools/data/adl-s-crb/industry.xml
+++ b/misc/config_tools/data/adl-s-crb/industry.xml
@@ -155,6 +155,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="2">
     <vm_type>POST_RT_VM</vm_type>
@@ -193,6 +194,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="3">
     <vm_type>POST_STD_VM</vm_type>
@@ -235,6 +237,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="4">
     <vm_type>POST_STD_VM</vm_type>
@@ -273,6 +276,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="5">
     <vm_type>POST_STD_VM</vm_type>
@@ -311,6 +315,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="6">
     <vm_type>POST_STD_VM</vm_type>
@@ -349,6 +354,7 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
   <vm id="7">
     <vm_type>KATA_VM</vm_type>
@@ -396,5 +402,6 @@
       <target_vm_id>1</target_vm_id>
       <target_uart_id>1</target_uart_id>
     </communication_vuart>
+    <PTM>n</PTM>
   </vm>
 </acrn-config>
-- 
2.25.1

